app/views/posts/index.html.erb

```diff
@@ -1 +1,13 @@
-<h1> List all user's posts</h1>
\ No newline at end of file
+<section>
+	<%= render 'shared/user', user: @user %>
+	<% @posts.each do |post| %>
+	  <%= render 'shared/post', post: post %>
+		<% post.most_recent_comments.each do | comment | %>
+			<% @user = User.find(comment.author_id) %>
+			<p><%= @user["name"] %>: <span><%= comment.text %></span></p>
+		<% end %>
+	<% end %>
+	<%= content_tag :div, class: "will-paginate-container" do %>
+	<%= will_paginate @posts %>
+	<% end %>	
+</section>
```

Comment:
-  [ ] [REQUIRED] Let's do some finishes also on the user's post index page after clucking the  **See all Post** button. Let's ensure it looks like the template provided.
### Your Design
<img width="1433" alt="Screenshot 2023-04-15 at 19 28 37" src="https://user-images.githubusercontent.com/58692590/232247260-be43f0d1-3d39-46b4-a185-bef5305a37ca.png">

### Design Template
<img width="463" alt="Screenshot 2023-04-15 at 19 22 31" src="https://user-images.githubusercontent.com/58692590/232247232-d0395861-c4b5-42f5-8a7a-7f66bb4ef1c0.png">